Latest releases lost support for Rust stable

From the readme:

> Comrak supports Rust stable.

This is not true of the latest rush of releases (0.7.1, 0.8.0, or 0.8.1). Here is the Clippy error:

```console
error[E0554]: `#![feature]` may not be used on the stable release channel
  --> src/lib.rs:74:37
   |
74 | #![cfg_attr(feature = "benchmarks", feature(test))]
   |                                     ^^^^^^^^^^^^^
```

Rustup informs me I'm running:

    stable-x86_64-unknown-linux-gnu unchanged - rustc 1.45.2 (d3fb005a3 2020-07-31)

Please either fix this so stable Rust can be used again or change the documentation to reflect the actual state of affairs (and maybe add a note about the change in the release notes for the sake of package builders).

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -36,4 +36,3 @@ timebomb = "0.1.2"
 
 [features]
 default = ["clap"]
-benchmarks = []

--- a/benches/progit.rs
+++ b/benches/progit.rs
@@ -1,0 +1,24 @@
+#![feature(test)]
+
+extern crate test;
+extern crate comrak;
+
+use comrak::{parse_document, format_html, Arena, ComrakOptions};
+use test::Bencher;
+
+#[bench]
+fn bench_progit(b: &mut Bencher) {
+    use std::fs::File;
+    use std::io::Read;
+
+    let mut file = File::open("script/progit.md").unwrap();
+    let mut s = String::with_capacity(524288);
+    file.read_to_string(&mut s).unwrap();
+    b.iter(|| {
+        let arena = Arena::new();
+        let root = parse_document(&arena, &s, &ComrakOptions::default());
+        let mut output = vec![];
+        format_html(root, &ComrakOptions::default(), &mut output).unwrap()
+    });
+}
+

--- a/src/lib.rs
+++ b/src/lib.rs
@@ -70,8 +70,6 @@
     unused_import_braces
 )]
 #![allow(unknown_lints, clippy::doc_markdown, cyclomatic_complexity)]
-#![cfg_attr(feature = "benchmarks", allow(unstable_features))]
-#![cfg_attr(feature = "benchmarks", feature(test))]
 
 extern crate entities;
 #[macro_use]

--- a/src/tests.rs
+++ b/src/tests.rs
@@ -1,7 +1,5 @@
 use cm;
 use html;
-#[cfg(feature = "benchmarks")]
-use test::Bencher;
 use timebomb::timeout_ms;
 use {parse_document, Arena, ComrakOptions};
 
@@ -60,23 +58,6 @@ macro_rules! html_opts {
             $(opts.$optclass.$optname = true;)*
         });
     };
-}
-
-#[cfg(feature = "benchmarks")]
-#[cfg_attr(feature = "benchmarks", bench)]
-fn bench_progit(b: &mut Bencher) {
-    use std::fs::File;
-    use std::io::Read;
-
-    let mut file = File::open("script/progit.md").unwrap();
-    let mut s = String::with_capacity(524288);
-    file.read_to_string(&mut s).unwrap();
-    b.iter(|| {
-        let arena = Arena::new();
-        let root = parse_document(&arena, &s, &ComrakOptions::default());
-        let mut output = vec![];
-        html::format_document(root, &ComrakOptions::default(), &mut output).unwrap()
-    });
 }
 
 #[test]

